src/homemade-ice-cream.md

```diff
@@ -0,0 +1,28 @@
+# Homemade Ice Cream
+
+_Easy Homemade Ice Cream_
+
+- ⏲️ Prep time: 5 min
+- 🍳 Cook time: 10 min
+- 💤 Rest time: 2-4 hours
+- 🍽️ Servings: 1 Serving
+
+## Ingredients
+
+- A Mason jar or some equivilent
+- 1 Cup Of Cream
+- 1½ drops of vanilla extract
+- 1½ Tablespoons of sugar
+- A small pinch of salt
```

Comment:
Salt/Pepper shouldn't be listed as ingredients.

I also noticed that it isn't listed in the directions. If there is a specific place that salt should be added, you can mention it in the directions, but no need for it in ingredients.